.env.sample

```diff
@@ -1,4 +1,4 @@
-FX_REMOTE_SETTINGS_WRITER_SERVER="https://settings-writer.prod.mozaws.net/v1"
+SERVER="https://remote-settings-dev.allizom.org/v1"
```

Comment:
Why are we using a dev server address here? What will send the update request to prod?

Reply:
adding the dev server URL here because it's a sample file, also, the address that was previously contained has been deprecated more than a year ago.

Container build from this repo gets the `SERVER` var injected during runtime by GKE, will point to remote-settings stage or prod accordingly.